code/modules/mob/mob.dm

```diff
@@ -209,7 +209,7 @@
 			else
 				client.perspective = EYE_PERSPECTIVE
 				client.eye = loc
-	return
+		return 1

```

Comment:
TRUE?

Reply:
What's the difference, functionally?

Am still new to this, sorry! <3

Reply:
TRUE/FALSE are for values that are treated like booleans, that is, you only care if they're strictly non-0 or 0. Actual numbers are used when you actually use the numeric output, like when returning an error code (the closest we come to that is `mob.stat`), or the value of a particular field. A function `isConscious()` would probably be expected to return TRUE/FALSE, but a function `getHealth()` is more likely to give proper numeric output.
To further the mob.stat example, there's lots of places in code where you care chiefly if a mob is conscious (`!stat == TRUE`), but you can also look at the value of stat numerically to tell whether the mob is unconscious or dead (`stat == UNCONSCIOUS` and `stat == DEAD`, respectively).

Reply:
This is done.